transit_odp/timetables/utils.py

```diff
@@ -244,10 +244,9 @@ def get_vehicle_journey_codes_sorted(
     """
     Get the vehicle journey codes sorted based on the departure time
     """
-
-    df_vehicle_journey_sorted = df_vehicle_journey_operating.groupby(
-        "vehicle_journey_code"
-    ).apply(lambda x: x.sort_values(by="departure_time"))
+    df_vehicle_journey_sorted = df_vehicle_journey_operating.sort_values(
+        by=["departure_day_shift", "departure_time"], ascending=[True, True]
```

Comment:
When departure_day_shift exist then the journey should be at the end of the timetable